core/frontend/src/components/notifications/NotificationCard.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <v-lazy
+    :options="{threshold: .5}"
+    transition="slide-x-transition"
+  >
+    <v-card elevation="0">
+      <v-list-item>
+        <v-list-item-icon>
+          <v-icon :color="level_color">
+            {{ level_icon }}
+          </v-icon>
+        </v-list-item-icon>
+
+        <v-list-item-content>
+          <v-list-item-title> {{ notification.service.name }} </v-list-item-title>
+          <v-list-item-subtitle> {{ notification.type }} - {{ notification.message }} </v-list-item-subtitle>
+          <v-list-item-subtitle>{{ time_since }} ({{ this.count }} times)</v-list-item-subtitle>
```

Comment:
The space between < and {{ is not consistent, can't we have eslint to detect that ?

Reply:
I couldn't found a rule to enforce for that :/